### ISSUE TITLE
Checkbox hidden behind other checkbox cannot be checked in VoiceOver on iOS

### ISSUE THREAD

**Issue by selfthinker** [alphagov/reported-bugs#40]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=198456
**Date:** 2019-06-01
**Reported by:** Anika Henke
**Related to:** ?

---

### Overview

When a checkbox is hidden behind another checkbox, it is impossible to tick or untick it with VoiceOver on iOS.

This works correctly with links or with the same checkbox in other screen readers, including VoiceOver on macOS.

### Steps to reproduce

1. Activate VoiceOver on iOS
2. Open Safari
3. Open https://jsbin.com/mekaxuz (reduced test case)
4. Navigate to the first (hidden) checkbox (labelled "Checkbox behind overlay")
5. Double tap to tick it

### Actual result

The checkbox stays unticked, VoiceOver says "unticked" and the visible checkbox on top of it (labelled "Checkbox inside overlay") gets ticked instead

### Expected result

The checkbox should get ticked, VoiceOver should say "ticked"
